docs/support-bundle-guide.md

```diff
@@ -0,0 +1,199 @@
+  fileServer:
+    url: sftp://yourtestdomain.com:22/root/test
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: # Empty list means all the ExternalNodes in the Namespace
+      - vm
+    namespace: vm-ns
+  fileServer:
+    url: yourtestdomain.com:22/root/test # Scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+You can check the status of `SupportBundleCollection` by running
+`kubectl get supportbundlecollections $Name-of-SupportBundleCollection -ojson`. The following example
+shows a successful creation of a` SupportBundleCollection`. `desiredNodes` shows the number of
+Nodes/ExternalNodes is expected in this request, while `collectedNodes` shows the number of
+Nodes/ExternalNodes which has already uploaded bundle files to the target file server.`collectedNodes`
+and `desiredNodes` should match the number of Nodes/ExternalNodes you want to collect suppor
+bundle if collection is running successfully.
+
+If following two conditions are presented, it means a bundle creation is succeeded,
+  1. "Completed" is true
+  2. "CollectionFailure" is false.
+
+If any expected Node/ExternalNode is failed to upload the bundle files in the required time, the
+"CollectionFailure" condition is marked as true.
+
+
+```bash
+$ kubectl get supportbundlecollections collect-support-bundle-for-kube-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+The output contains three tarballs. To access these files, please SSH into the ftp server
+`yourtestdomain.com` with port 22 or scp them to any other storages you used in CR. There will be
+two tarballs for `collect-support-bundle-for-kube-nodes` and one for `collect-support-bundle-for-vm-external-node`
```

Comment:
two tarballs for `collect-support-bundle-for-kube-nodes` ( "collect-support-bundle-for-kube-nodes_kube-master.tar.gz" and "collect-support-bundle-for-kube-nodes_kube-worker.tar.gz" ) and one for `collect-support-bundle-for-vm-external-node` ( "collect-support-bundle-for-vm-external-node_vm.tar.gz") showing in `/root/test` folder.

Reply:
updated